app/routes/socialEvents.tsx

```diff
@@ -0,0 +1,15 @@
+import EventCard from '~/components/common/eventCard';
+
+export default function socialEvents() {
+  return (
+    <>
+      <EventCard
+        date={30}
+        month="AUG"
+        time="12AM to 1:30PM"
+        eventTitle="React India Conf"
+        participantCount="100 Participants"
```

Comment:
I guess we would be getting a number here and you can append the string in the component itself based on the count

Reply:
Got it, updating it 
Thanks!

Reply:
Updated to number type and appended it to string in the component. 
Thanks